doc/DeveloperGuide.md

```diff
@@ -85,6 +85,26 @@ Use case ends.
 
 3a. The given index is invalid
 
+> 3a1. AddressBook shows an error message <br>
+  Use case resumes at step 2
+
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
```

Comment:
Can consider combining the first two into one. So as to capture the interaction between User and AddressBook line by line.